packages/debugger/test/session.spec.ts

```diff
@@ -73,7 +73,9 @@ describe('Debugger.Session', () => {
       });
       await debugSession.start();
       await debugSession.stop();
-      expect(events).toEqual(['output', 'initialized', 'process']);
+      expect(events).toEqual(
+        expect.arrayContaining(['output', 'initialized', 'process'])
+      );
```

Comment:
This makes the tests a bit more flexible as other event messages might appear in this sequence.